lua/telescope/builtin/init.lua

```diff
@@ -888,6 +888,43 @@ builtin.keymaps = function(opts)
   }):find()
 end
 
+builtin.highlights = function(opts)
+  opts = opts or {}
+
+  local highlights = vim.fn.getcompletion('', 'highlight')
+
+  pickers.new({}, {
+    prompt_title = 'Highlights',
+    finder = finders.new_table {
+      results = highlights,
+      entry_maker = function(entry)
+        return {
+          value = entry,
+          ordinal = entry,
+          display = entry,
+          preview_command = function(bufnr)
+            vim.api.nvim_buf_set_option(bufnr, 'filetype', 'vim')
+            local output = vim.split(vim.fn.execute('hi ' .. entry), '\n')
```

Comment:
you can probably use things like `nvim_get_hl_by_name` or similar to get some of this information.

Reply:
I don't think this is useful for the previewer. This output will be 100% in the previewer and i think it is better if we present the user with something like this rather than something like
```lua
{
  foreground = 16711680,
  bold
}
```